public/app/plugins/datasource/influxdb/components/InfluxLogsQueryField.tsx

```diff
@@ -77,6 +81,11 @@ export class InfluxLogsQueryField extends React.PureComponent<Props, State> {
     );
 
     this.props.onChange(queryModel.target);
+
+    // Only run the query if measurement, field, and pairs are all set
+    if (this.pairsSet(pairs) && measurement && field) {
```

Comment:
Selection of measurement and field should trigger query to be executed. Filters should be optional.

Reply:
Sure, will change